### DISCUSSION TITLE
Caprover docker App deployment file ?

### DISCUSSION THREAD

**Discussion by rdiok3** [caprover/caprover#1999] [Q&A]
Dear Commmunity,

I am trying to build a deployment dockerfile for a django app,
in such a way that the dockerfile internally sets as  gunicorn
server port the one selected through the Caprover graphical
app interface.

Right now the dockerfile has hardcoded the port 80 as gunicorn
port. How can I give/code the instruction to make the dockerfile 
use the port set through the Caprover graphical interface ?

I am ataching an image explaining what I mean.

Thank you very much for your help !

![CaproverQuestion](https://github.com/caprover/caprover/assets/109904698/78adeb00-09f7-4f4a-8b37-6bb4cff9325a)

**Comment by githubsaturn** [comment]
These are two different things. The port that you see in CapRover interface is the port that nginx tries to connect to. 

In theory, you can have many many ports in your Dockerfile, but you have specify one of them for the nginx to connect to.
